core/src/main/kotlin/org/evomaster/core/database/SqlInsertBuilder.kt

```diff
@@ -86,7 +90,9 @@ class SqlInsertBuilder(
                         unique = c.unique,
                         lowerBound = lowerBound,
                         upperBound = upperBound,
-                        enumValuesAsStrings = enumValuesAsStrings
+                        enumValuesAsStrings = enumValuesAsStrings,
+                        similarToPatterns = similarToPatterns,
+                        likePatterns = likePatterns
                 )
```

Comment:
i have seen LIKE and SIMILAR TO (plus other kinds of regexs) to be different among databases... :(
i guess that somehow we need to store info of which database we are using

Reply:
I am adding databaseType to the column, so it flows to the RegexHandler.